apps/base/traefik-hub/release.yaml

```diff
@@ -9,7 +9,7 @@ spec:
   chart:
     spec:
       chart: traefik-hub
-      version: v2.7.0
+      version: v2.9.0
```

Comment:
We need to fix the point of `IngressClass` first.

Reply:
Added a fix